wordpress-mobile/WordPress-iOS
# Moved MediaServiceRemote classes, models, and related tests to WordPressKit

Fixes #7312  and #7313  

This PR:
* Migrates the `RemoteMedia` model to the WPKit framework
* Migrates `MediaServiceRemoteREST` to the WPKit framework
* Migrates `MediaServiceRemoteXMLRPC` to the WPKit framework
* Migrates `MediaServiceRemoteRESTTests.swift` to the WPKit framework
* Migrates `WordPressComRestApiTests.swift` to the WPKit framework
* Migrates `WordPressOrgXMLRPCApiTests.swift` to the WPKit framework

Ref #7301

**To test:**
1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green.
5. Build & run the app:
a. verify media management is working as expected on .com
b. verify media management is working are working as expected on .org

Needs review: @astralbodies
